client/homescreen/activity-panel/index.js

```diff
@@ -25,29 +26,50 @@ import { getAllPanels } from './panels';
 import { getUnapprovedReviews } from './reviews/utils';
 
 export const ActivityPanel = () => {
-	const panelsData = useSelect( ( select ) => {
-		const totalOrderCount = getSetting( 'orderCount', 0 );
-		const orderStatuses = getOrderStatuses( select );
-		const reviewsEnabled = getSetting( 'reviewsEnabled', 'no' );
-		const countUnreadOrders = getUnreadOrders( select, orderStatuses );
+	const settingsData = useSelect( () => {
```

Comment:
@octaedro is there a particular reason you split up the `useSelect` into 3 different one's?
Not a huge issue, but I believe keeping a single `useSelect` for all the data is more efficient then multiple `useSelects`.

Reply:
Hi @louwie17, thank you for your review.
As I was consulting 3 different stores I thought it would be a good idea to use different `useSelect`s. So if the reviews store changes we won't call the other two selectors on the other stores (that won't return anything new).
The difference is pretty small and I'm totally ok with the change you suggested since it makes this code easier to read. Please let me know if I'm missing something.

Reply:
> So if the reviews store changes we won't call the other two selectors on the other stores (that won't return anything new).

Unfortunately that's not the case, you did have to get the store object (I think returned by the `register` function, although that's not common practice).
The `useSelect` is triggered when any registered store updates, so in this case all three `useSelect` will get triggered if either the reviews store, or any of the other registered stores are updated.

Reply:
>So if the reviews store changes we won't call the other two selectors on the other stores (that won't return anything new).

I think thats ok because selector results are cached based on supplied arguments. Lets fix this up before merge

Reply:
It seems like a single useSelect is slightly faster unless they are from different stores, in which case its slightly faster, but will probably change. So your choice @octaedro. I'm ok with either.

p7H4VZ-2Q6-p2

Reply:
Ok, sounds good to me 👍
This change was added in the commit 69f1809

Reply:
@octaedro re-tested it locally and all is good, although one odd thing with the logic, is that the `Orders` panel will be displayed, but the reviews/stock panels won't until the task list is hidden.